Makefile

```diff
@@ -94,10 +94,6 @@ ifeq ($(REMOTE_INTEGRATION),true)
 endif
 	REMOTE_INTEGRATION=$(REMOTE_INTEGRATION) go test -v $(REPOPATH)/integration -timeout 10m
 
-.PHONY: coverage
```

Comment:
Can we still keep this in? 
Its used for code coverage i guess. 
https://github.com/GoogleContainerTools/skaffold/blob/master/.travis.yml#L14

Reply:
Looks like you removed it .travis.yaml but we need it for https://github.com/codecov/example-go plugin.

Reply:
`make test` is supposed to produce the same output, now.

Reply:
I mean `make test` is supposed to produce the coverage file that codecov needs.